react/components/BaseSpecificationBadges.tsx

```diff
@@ -136,7 +143,9 @@ const BaseSpecificationBadges: StorefrontFunctionComponent<Props & BaseProps> =
         const { displayValue } = badge
         let valueToShow = displayValue
         if (displayValue === DisplayValues.specificationValue) {
-          valueToShow = badge.specification.values[0]
+          valueToShow = badge.specification.values.join(
+            specificationValueSeparator
+          )
```

Comment:
Should we add blank space around the separator?

Reply:
Not necessarily, because it's as simple as adding it to the prop 🤔

Reply:
true